src/views/TicketsView/Prices/Prices.jsx

```diff
@@ -21,185 +21,199 @@ import {
-            <TicketTitle color={colors.gris90}>Billet entreprise</TicketTitle>
-            {/* <TicketBadge>Complet</TicketBadge> */}
-
-            <TicketPrice>905$</TicketPrice>
-
-            <Button
-              to='https://ti.to/web-a-quebec/waq24/fr'
-              tag='href'
-              small
-              animated
-              rel='noopener noreferrer'
-              target='_blank'
-              css={buttonStyle}
-            >
-              Acheter mon billet
-            </Button>
-
-            <TicketText>
-              <p>
-                Tu veux assister aux conférences du matin et passer le relais à
-                ta collègue en pm? C’est le billet qu’il vous faut!
-              </p>
-              <ul>
-                <li>3 journées de conférences;</li>
-                <li>
-                  6 accès pour écouter les contenus de la salle principale en
-                  rediffusion;
-                </li>
-                <li>les meilleurs 5 à 7 en ville;</li>
-                <li>tes repas et boissons durant les 3 jours;</li>
-                <li>
-                  une gestion des billets pour faciliter la cueillette de ta
-                  cocarde.
-                </li>
-              </ul>
-            </TicketText>
-          </Center>
-        </Paper>
-
-        <Paper
-          lightColor={colors.peach}
-          darkColor={colors.blueberry10}
-          rounded
-          outlined
-          elevation={elevation.none}
-          css={blockContainerStyle}
-        >
-          <Center maxWidth='320px' intrinsic withText css={ContentContainer}>
-            <TicketTitle color={colors.gris90}>Billet étudiant</TicketTitle>
-            {/* <TicketBadge>Complet</TicketBadge> */}
-
-            <TicketPrice>300$</TicketPrice>
-
-            <Button
-              to='https://ti.to/web-a-quebec/waq24/fr'
-              tag='href'
-              small
-              animated
-              rel='noopener noreferrer'
-              target='_blank'
-              css={buttonStyle}
-            >
-              Acheter mon billet
-            </Button>
-
-            <TicketText>
-              <p>Te former, réseauter et 5à7ter. Quoi demander de plus?</p>
-              <ul>
-                <li>un billet à un prix réduit;</li>
-                <li>3 journées de conférences;</li>
-                <li>tes repas et boissons durant les 3 jours;</li>
-                <li>
-                  un accès pour écouter les contenus de la salle principale en
-                  rediffusion;
-                </li>
-                <li>les meilleurs 5 à 7 en ville.</li>
-              </ul>
-            </TicketText>
-          </Center>
-        </Paper>
-      </BlocksWrapper>
-    </Switcher>
-  </Center>
-);
+const Prices = () => {
+  const [isRegularPrice, setIsRegularPrice] = useState(true);
+
+  useEffect(() => {
+    const currentDate = new Date();
```

Comment:
Change pour
```suggestion
    const currentDate = new Date().getTime();
    const targetDate = new Date('2024-03-21').setHours(24, 0, 0, 0) // set pour minuit
```
et tu vas être correct